test/e2e/tests/dapp-interactions/block-explorer.spec.js

```diff
@@ -82,30 +79,30 @@ describe('Block Explorer', function () {
         await driver.clickElement(
           '[data-testid="account-overview__asset-tab"]',
         );
-        const [, tst] = await driver.findElements(
-          '[data-testid="multichain-token-list-button"]',
-        );
-        await tst.click();
+
+        await driver.clickElement({
+          text: 'TST',
+          tag: 'span',
+        });
+
         await driver.clickElement('[data-testid="asset-options__button"]');
         await driver.clickElement({
           text: 'View Asset in explorer',
           tag: 'div',
         });
 
         // Switch to block explorer
-        await driver.waitUntilXWindowHandles(2);
-        const windowHandles = await driver.getAllWindowHandles();
-        await driver.switchToWindowWithTitle('E2E Test Page', windowHandles);
-        const body = await driver.findElement(
-          '[data-testid="empty-page-body"]',
-        );
+        await driver.switchToWindowWithTitle('E2E Test Page');
 
         // Verify block explorer
-        assert.equal(await body.getText(), 'Empty page by MetaMask');
-        assert.equal(
-          await driver.getCurrentUrl(),
-          'https://etherscan.io/token/0x581c3C1A2A4EBDE2A0Df29B5cf4c116E42945947',
-        );
```

Comment:
this is an anti-pattern: we should wait for the value we want, instead of grabbing the selector and then asserting it has the correct value, as it can be the case the value is not yet loaded